book/content/reproducibility/04/barriers.md

```diff
@@ -18,7 +18,9 @@ This chapter outlines some of those barriers, and a couple of suggestions to get
 
 ## Barrier 2
```

Comment:
This formats the barrier itself as the heading (rather than having the heading say "Barrier 2" and then reading down to know what it is. I think it will make the chapter easier to scan if someone's quickly reading through.
```suggestion
## Publication bias towards novel findings
```